test/function/facility.cpp

```diff
@@ -59,6 +61,8 @@ namespace {
     STUMPLESS_FOREACH_FACILITY( CHECK_FACILITY_ENUM )
   }
 
+//Old test
+/*
```

Comment:
don't comment out code -  in this case the old test should be able to remain, as it should continue to pass.